bigbio/biodatasets/quaero/quaero.py

```diff
@@ -225,86 +180,39 @@ def _split_generators(self, dl_manager):
 
     # method parameters are unpacked from `gen_kwargs` as given in `_split_generators`
     def _generate_examples(self, filepath, split):
-
         if self.config.subset_id == "quaero_emea":
             subset = "EMEA"
         elif self.config.subset_id == "quaero_medline":
             subset = "MEDLINE"
 
-        key = 0
+        folder = Path(filepath) / "QUAERO_FrenchMed" / "corpus" / split / subset
+
         if self.config.schema == "source":
-            file = f"{subset}_{split}_bioc"
-            f = os.path.join(filepath, "QUAERO_BioC", "corpus", split, file)
-            reader = bioc.biocxml.BioCXMLDocumentReader(str(f))
-
-            for xdoc in reader:
-                doc_text = self._get_document_text(xdoc)
-                yield (
-                    key,
-                    {
-                        "id": xdoc.id,
-                        "document_id": xdoc.id,
-                        "entities": [
-                            self._get_bioc_entity(span, doc_text)
-                            for passage in xdoc.passages
-                            for span in passage.annotations
-                        ],
-                        "passages": [
-                            {
-                                "id": xdoc.id,
-                                "type": passage.infons,
-                                "text": [passage.text],
-                                "offsets": [
-                                    self._get_bioc_entity(span, doc_text)["offsets"][0]
-                                    for span in passage.annotations
-                                ],
-                            }
-                            for passage in xdoc.passages
-                        ],
-                    },
-                )
-                key += 1
+            for guid, txt_file in enumerate(folder.glob("*.txt")):
```

Comment:
I'm a little concerned that `glob` will not return a deterministic order for the files. Is there a sort key on the file names that makes sense?

Reply:
The file name seems to be some kind of document id, so sorting by file name would be an option.
To be honest, I just copied that part from [examples/mlee.py](https://github.com/bigscience-workshop/biomedical/blob/master/examples/mlee.py), as this was given as a reference in the contribution guide.

>BRAT: Use [our custom brat parser](https://github.com/bigscience-workshop/biomedical/blob/master/bigbio/utils/parsing.py). Example usage can be found in [examples/mlee.py](https://github.com/bigscience-workshop/biomedical/blob/master/examples/mlee.py).

Reply:
ok, looks good, we can update in another PR if there turns out to be a canonical order that we want to enforce.  
@leonweber @sg-wbi @WojciechKusa I think we should probably update that example ^